src/scripts/temporaryBattle/TemporaryBattleList.ts

```diff
@@ -75,7 +75,12 @@ TemporaryBattleList['Cue Ball Paxton'] = new TemporaryBattle(
         new TemporaryBattleRequirement('Biker Goon 3'),
     ]
 );
-
+TemporaryBattleList['Mime Interview'] = new TemporaryBattle(
+    'Mime Interview',
+    [new GymPokemon('Mr. Mime', 160225334, 100)],
```

Comment:
Temporary battles are sorted by when we expect the players to defeat them, not where they show up.
So because we expect players to complete them in early kalos, they should all be listed as the first, in kalos